corehq/motech/repeaters/_design/views/repeat_records/map.js

```diff
@@ -10,5 +10,7 @@ function (doc) {
         }
         emit([doc.domain, doc.repeater_id, state, doc.last_checked], null);
         emit([doc.domain, null, state, doc.last_checked], null);
+        emit([doc.domain, null, null, doc.last_checked], null);
+        emit([doc.domain, doc.repeater_id, null, doc.last_checked], null);
```

Comment:
That's right. We don't track each attempt in Couch, we only have the latest tried date (for SQL, I have used `SQLRepeatRecordAttempt.created_at` not `SQLRepeatRecord.registered_at`). So this filters records based on that (it will capture both succeeded and retried records in the given range).